### PR TITLE
Github Enterprise Support

### LINKED ISSUE #59: github enterprise
afaik this should just be providing an api prefix.

Need to find a running instance to test this against though 🤔

### PR DESCRIPTION
**Pull request by gaborbernat**
Resolves #59

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 /.tox
 /venv*
 .vscode
+.tox
```
Comment (asottile) [inline .gitignore]: hmm that's weird, shouldn't line 6 have handled this?

Comment (gaborbernat) [reply .gitignore]: Hmm maybe I've overlooked something 🤔

Comment (gaborbernat) [reply .gitignore]: I can do Monday 😎but just pass the base URL to the API within the config.

**Comment by gaborbernat** [comment]
@asottile I've addressed your comments now 👍

**Pull request merged**